x-pack/plugins/cases/server/client/factory.ts

```diff
@@ -106,6 +108,7 @@ export class CasesClientFactory {
       userActionService: this.options.userActionService,
       logger: this.logger,
       authorization: auth,
+      auditLogger,
```

Comment:
Why don't we use our audit logger? (`auditLogger: new AuthorizationAuditLogger(auditLogger),`)

Reply:
Definitely could, see above comment.